bin/run-addon

```diff
@@ -165,8 +165,10 @@ run_jpm() {
   echo "Running Firefox.  Reloading with jpm *will not* happen"
   echo "If the Screenshots icon doesn't show up go to:"
   echo "  Tools > Add-ons > Extensions and look for an error loading the add-on"
+  settings_file="./build/.jpm-settings.json"
+  echo '{"extensions.legacy.enabled": true}' > "$settings_file"
```

Comment:
Since we're in here, do you want to also set `xpinstall.signatures.required` to false?